build_projects/update-dependencies/Config.cs

```diff
@@ -82,7 +82,8 @@ private static string GetDefaultUpstreamBranch()
 
         private static string GetDefaultCoreSetupVersionFragment()
         {
-            string coreSetupChannel = GetRepoMSBuildPropValue("BundledRuntimes.props", "CoreSetupChannel") ?? "master";
+            // by default, the current core-setup branch should match the current cli branch name
```

Comment:
Anyone can override this string with the `CORESETUP_VERSION_FRAGMENT` env variable.  So we don't need to make code changes in order to change this.

This is just the default case, when someone (i.e. the Maestro subscriptions.json file) doesn't set that env var.